samples/scala-reliable-timers/src/test/scala/com/example/actions/OrderActionSpec.scala

```diff
@@ -0,0 +1,52 @@
+package com.example.actions
+
+import com.example.OrderRequest
+import com.example.OrderStatus
+import com.google.protobuf.empty.Empty
+import kalix.scalasdk.action.Action
+import kalix.scalasdk.testkit.ActionResult
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+// This class was initially generated based on the .proto definition by Kalix tooling.
+//
+// As long as this file exists it will not be overwritten: you can maintain it yourself,
+// or delete it so it is regenerated as needed.
+
+class OrderActionSpec
+    extends AnyWordSpec
+    with Matchers {
+
+  "OrderAction" must {
```

Comment:
The annoying thing about the Action testkit is that we can't use `components` and `timers`, but this Action use it a lot. 

Now we generate the tests skeleton for the user, but they don't have the means to implement it and they will have to keep this file because of the codegen.

Reply:
Yeah, timers will be easy to do in the testkit as well, as they can just be a stored list of scheduled timers to assert happened or not after a call. I don't think that should be held up by missing support for async.exec-calls, which are quite different, not having a solution.

That would mean a dummy timer service impl in the testkit action runner with access through the ActionTestKit

Reply:
I expect we will add testkit support before releasing the SDK with the timers feature? I created https://github.com/lightbend/kalix-jvm-sdk/issues/957 to track that